lib/config/builder/list.ts

```diff
@@ -1,31 +1,8 @@
 import { BuilderExtract, IBuilder } from "./builder";
 import { Config } from "./config";
-import { InputSpec, ListValueSpecNumber, ListValueSpecString, UniqueBy, ValueSpecList } from "../config-types";
+import { InputSpec, ListValueSpecString, UniqueBy, ValueSpecList } from "../config-types";
 import { guardAll } from "../../util";
-import { range } from "lodash";
 
-/**
- * Used as a subtype of Value.list
-```ts
```

Comment:
Um, we need the docs, maybe update this example from the output.ts that gets generated